### ISSUE TITLE
Image title/alt-tag

### ISSUE THREAD

**Issue by patrickdemooij9** [patrickdemooij9/SeoToolkit.Umbraco#236]
Would be great if we could find a way to add this as a new functionality. The ability to have a title/alt-tag per image. Ofcourse you can currently do that with compositions/image types, but would be nice if it works out of the box.

**Comment by DanielOberlechner** [comment]
You mean that you can set the alt tag per image in the media section of UmbracoCMS? That would be awesome I think! Would bring a lot of improvement to developing! Sounds like a good idea! :) 👍

**Comment by patrickdemooij9** [comment]
Yes, that is the plan. Probably have an extra tab that allows you to setup the alt tag per image

**Comment by DanielOberlechner** [comment]
Maybe you want to keep in mind that some people use multiple languages with UmbracoCMS and so it would make sense to  allow to set titles in multiple languages ... just a reminder ..
